modules/openapi-generator/src/main/resources/php-slim4-server/README.mustache

```diff
@@ -19,7 +19,7 @@ This server has been generated with [Laminas (Zend) PSR-7 implementation](https:
 ## Requirements
 
 * Web server with URL rewriting
-* PHP 7.2 or newer
+* PHP 7.3 or newer (8.0 compatible, but not tested yet)
```

Comment:
I suggest remove `(8.0 compatible, but not tested yet)` from the auto-generated readme and put the information in this PR instead.

Reply:
Cool, I'll change it in a few minutes. The problem is I already wrote the same in #9986:
```diff
@@ -12,7 +12,8 @@ For more information, please visit [{{{infoUrl}}}]({{{infoUrl}}}).

--- PHP 7.2 and later.
+++ PHP 7.3 and later.
+++ Should also work with PHP 8.0 but has not been tested.
```